Doc/glossary.rst

```diff
@@ -1017,17 +1031,21 @@ Glossary
       :func:`bytes.splitlines` for an additional use.
 
    variable annotation
-      A type metadata value associated with a module global variable or
-      a class attribute. Its syntax is explained in section :ref:`annassign`.
-      Annotations are stored in the :attr:`__annotations__` special
-      attribute of a class or module object and can be accessed using
-      :func:`typing.get_type_hints`.
+      An :term:`annotation` of a global variable, or class variable.
+
```

Comment:
I think this empty line is not needed

Reply:
I think its best to separate a concise description of the term from the example.

If you still think it's best to put them in the same paragraph, I'll make it so :)